esmvaltool/recipes/recipe_shapeselect_py.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+datasets:
+  - {dataset: EC-EARTH, project: CMIP5, mip: Amon, exp: historical, ensemble: r12i1p1, start_year: 1990, end_year: 1999}
+#  - {dataset: CRU, mip: Amon, project: OBS, type: reanaly, version: 1, start_year: 1990, end_year: 1994, tier: 3}
+ #  - {dataset: bcc-csm1-1-m,  project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1998,  end_year: 1999}
+ 
+
+preprocessors:
+  preprocessor_1: {}
```

Comment:
There is no need to define an empty preprocessor, just leave it out, also in the variables section.